04/app.js

```diff
@@ -1,5 +1,35 @@
 document.addEventListener('DOMContentLoaded', init);
 
 function init() {
+    const form = document.querySelector(".form");
+    form.addEventListener("submit", (event) => {
+        //const latInput = document.querySelector(".form__field--lat");
+        //const lat = lat.value;
+        //console.log(lat)
+        event.preventDefault()
+        const latInput = document.querySelector(".form__field--lat");
+        const lngInput = document.querySelector(".form__field--lng");
+        const lng = lngInput.value;
+        const lat = latInput.value;
+        console.log(lat)
+        console.log(lng)
+        const strongFirst = document.querySelector(".weather__lat");
+        const strongSec = document.querySelector(".weather__lng");
+        let weatherSum = document.querySelector(".weather__summary");
+        let weatherTemp = document.querySelector(".weather__temperature");
+        strongFirst.innerText = lat;
+        strongSec.innerText = lng;
+        
+        const promise = fetch(`https://api.weatherbit.io/v2.0/current?key=832e544e55c84199b5277249cedeb143&lang=pl&lat=${lat}&lon=${lng}`);
+    
+        promise
+        .then(response => response.json())
+        //.then(d => console.log(d.data[0].weather.description))
```

Comment:
Jeśli pytasz o tą linię, to ona nie zwraca obietnicy i nie ma czego obsłużyć `.then`, bo tutaj zwracasz to co zwróci `console.log()` czyli `undefined`.